drivers/Makefile.include

```diff
@@ -118,6 +118,14 @@ ifneq (,$(filter ina220,$(USEMODULE)))
   USEMODULE_INCLUDES += $(RIOTBASE)/drivers/ina220/include
 endif
 
+ifneq (,$(filter ina3221,$(USEMODULE)))
+  USEMODULE_INCLUDES += $(RIOTBASE)/drivers/ina3221/include
+endif
+
+ifneq (,$(filter ina3221_with_alerts,$(USEMODULE)))
+  CFLAGS += -DINA3221_WITH_ALERTS=1
```

Comment:
This is not needed. You can test for the presence of the pseudo module `ina3221_with_alterts` by

```C
#ifdef MODULE_INA3221_WITH_ALERTS
```